src/main/java/io/seqera/tower/cli/responses/RunCreated.java

```diff
@@ -0,0 +1,17 @@
+package io.seqera.tower.cli.responses;
+
+public class RunCreated extends Response {
```

Comment:
I think that you can reuse the `RunSubmited` response that uses the launch command.